File: frontend/src/media.js
export const IMAGE_FILE_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.bmp', '.webp', '.avif'];

export const IMAGE_FILE_ACCEPT = IMAGE_FILE_EXTENSIONS.join(',');

export const isSupportedImageFile = (file) => {
  if (!file) return false;
  const name = String(file.name || '').trim().toLowerCase();
  return IMAGE_FILE_EXTENSIONS.some((ext) => name.endsWith(ext));
};

File: src/web/constants/media.py
"""跨上下文共享的媒体与数据集基础常量。"""

IMAGE_FILE_EXTENSIONS = (".jpg", ".jpeg", ".png", ".bmp", ".webp", ".avif")
VIDEO_FILE_EXTENSIONS = (".mp4", ".avi", ".mov", ".mkv", ".webm")
VIDEO_FILE_EXTENSION_SET = frozenset(VIDEO_FILE_EXTENSIONS)
DATASET_SPLITS = ("train", "val", "test")
EVAL_SPLITS = ("val", "test")
ROBOFLOW_CONFIG_FILENAMES = ("data.yaml", "data.yml")
